pkg/render/stats.go

```diff
@@ -0,0 +1,104 @@
+package render
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/chainguard-dev/bincapz/pkg/bincapz"
+	"github.com/chainguard-dev/bincapz/pkg/report"
+)
+
+func riskLevelStatistics(files map[string]bincapz.FileReport, riskMap map[int][]string, riskPercentages map[int]float64) []bincapz.Kv {
+	for path, rf := range files {
+		riskMap[rf.RiskScore] = append(riskMap[rf.RiskScore], path)
+		for riskLevel := range riskMap {
+			riskPercentages[riskLevel] = (float64(len(riskMap[riskLevel])) / float64(len(files))) * 100
+		}
+	}
+
+	var riskLevelStatistics []bincapz.Kv
```

Comment:
Can you use a shorter name for this? Maybe `rls` or `stats`? 

In Go, the length of a variable name indicates the size of its scope, and this variable is only used for 8 lines. The name is also the same as the function it resides in, which is unusual and causes a stutter pattern: https://go.dev/wiki/CodeReviewComments#variable-names

Reply:
Appreciate the context!

Addressed in [`5ad17ab` (#121)](https://github.com/chainguard-dev/bincapz/pull/121/commits/5ad17abd03cd920455ed146562cb0bf3c9ac992f).